js/job.js

```diff
@@ -1,26 +1,19 @@
+(function(context){
+  context.jobs = [];
 
-var jobs = [];
+  function Job (opt) {
+    this.company = opt.company;
+    this.position = opt.position;
+    this.dates = opt.dates;
+    this.responsibilities = opt.responsibilities;
+  }
 
-function Job (opt) {
-  this.company = opt.company;
-  this.position = opt.position;
-  this.dates = opt.dates;
-  this.responsibilities = opt.responsibilities;
-}
-
-var retrieveJobHistory = function(){
-  $.getJSON('https://raw.githubusercontent.com/tetsaiga86/Portfolio/class-07/data/jobHistory.json', function(data){
-    localStorage.jobHistory = JSON.stringify(data);
-    data.forEach(function(ele) {
-      jobs.push(new Job(ele));
+  context.retrieveJobHistory = function(){
+    $.getJSON('https://raw.githubusercontent.com/tetsaiga86/Portfolio/class-07/data/jobHistory.json', function(data){
+      localStorage.jobHistory = JSON.stringify(data);
+      data.forEach(function(ele) {
+        jobs.push(new Job(ele));
+      });
     });
-  });
-};
-
-//for future project list
-// projectHistory.forEach(function(ele) {
-//   projects.push(new Project(ele));
-// });
-// projects.forEach(function(obj) {
-//   $('#projects').append(obj.toHtml());
-// });
+  };
```

Comment:
You're not attaching context to the window here, so while it works when you have contexts to attach to, it makes things kind of weird with the window. I expect this to result in weird problems down the line.